tests/run/sys_path_globbed.srctree

```diff
@@ -0,0 +1,27 @@
+# tag: gh5942
+
+PYTHON setup.py build_ext --inplace
+PYTHON -c "from tglob import mytest; mytest()"
+
+######## tglob.pyx ########
+def mytest():
+    assert True
+
+######## setup.py ########
+import sys
+
+from Cython.Build.Dependencies import cythonize
+from distutils.core import setup, Extension
+
+sys.path.append("[Hello-World]*?")
+
+try:
+    from __builtin__ import reload
+except ImportError:
+    from importlib import reload
+reload(sys)  # Magic
```

Comment:
I couldn't find any difference keeping this code. And it seems weird to have it.
```suggestion
```